src/routes/MyWork/components/WorkList/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import { useSelector } from "react-redux";
-import WorkItem from "../WorkItem";
+
+import { WorkList as WorkListTs } from '../../../../../src-ts/tools/work'
```

Comment:
I'm not sure I understand what a `WorkListTs` is -- can we name this any better?

Reply:
yeah--i struggled with that as well. WorkListTypescript? WorkListNextGen? something like that?

Reply:
Is there a reason it can't just be `WorkList` -- do we have a naming collision for an existing type that we are trying to replace with the Typescript version?